docs/latin.rst

```diff
@@ -721,6 +721,44 @@ The StringUtils module contains utility methods for processing scansion and text
    Out[2]: 'I m ok  Oh              Fine'
 
 
+Semantics
+=========
+The Semantics module allows for the lookup of Latin lemmata, synonyms, and translations into Greek. Lemma, synonym, and translation dictionaries are drawn from the open-source `Tesserae Project<http://github.com/tesserae/tesserae>`
+
```

Comment:
Also mention that we are storing this data in `https://github.com/cltk/latin_models_cltk/tree/master/semantics` and `https://github.com/cltk/greek_models_cltk/tree/master/semantics`. And explain that the repos (or just Latin for this one?) need to be imported first (give link to http://docs.cltk.org/en/latest/importing_corpora.html#importing-a-corpus … we often get Issues raised b/c people don't know what functionality requires a data dir, so this is a good, practical help).